modules/ob-installing-buildpacks-v3-strategy.adoc

```diff
@@ -0,0 +1,29 @@
+// This module is included in the following assembly:
+//
+// builds/installing-sample-build-strategies.adoc
+
+:_content-type: PROCEDURE
+[id="installing-buildpacks-v3-strategy_{context}"]
+= Installing buildpacks v3 strategy
+
+You can install the `buildpacks v3` strategy at your namepsace or cluster level. By installing the `buildpacks v3` strategy at cluster level, you can share it across different namespaces within your cluster.
```

Comment:
Same comment as above: " You can share the buildkit strategy across different namespaces within your cluster by installing it at the cluster level."